scss/civicrm/event/_detail.scss

```diff
@@ -0,0 +1,243 @@
+    padding-right: 15px;
+  }
+
+  fieldset#discount,
+  fieldset#map-field {
+    padding-left: 20px;
+    padding-right: 20px;
+
+    > legend {
+      @include section-title;
+      display: block;
+      width: 100%;
+      margin-left: -20px;
+    }
+
+    table {
+      min-width: 100%;
+    }
+
+    tr {
+      background-color: transparent;
+      border-top: 1px solid $crm-table-border-color;
+      display: block;
+      margin: 0 -20px;
+      padding: 0px 20px 0px 20px;
+    }
+
+    td {
+      border: 0;
+      display: inline-block;
+      padding: 5px 10px 9px;
+      width: 260px;
+    }
+
+    .dateplugin {
+      margin-top: 5px;
+    }
+
+    .columnheader {
+      display: block;
+      background-color: $gray-lighter;
+      border-bottom: 1px solid $crm-grayblue-dark;
+      border-left: 0;
+      border-radius: 0;
+      border-right: 0;
+      margin: 20px -20px 0;
+      padding: 6px 20px 4px 20px;
+
+      br {
+        display: none;
+      }
+
+      td {
+        color: $crm-columnheader-title-color;
+        font-size: 13px;
+      }
+
+      label {
+        margin-left: 5px;
+      }
+    }
+  }
+
+  fieldset#discount {
+    .columnheader td:first-child,
+    .form-item td:first-child {
+      padding-left: 2px;
+      width: 20px;
+    }
+
+    .crm-form-submit {
+      margin: 10px 20px 0;
+    }
+  }
+
+  #isDiscount {
+    padding-left: 20px;
+    padding-right: 20px;
+
+    .crm-event-manage-fee-form-block-is_discount td {
+      width: auto;
+    }
+  }
+
+  .crm-collapsible {
+    padding: 20px;
+
+    &.collapsed .collapsible-title {
+      box-shadow: $crm-accordion-shadow;
+
+      &:before {
+        content: $fa-chevron-right;
+      }
+    }
+
+    .collapsible-title {
+      background-color: $crm-collapsible-title-background;
+      background-image: none;
+      border: 0;
+      color: $crm-pure-white-color;
+      font-size: 18px;
+      font-weight: 600;
+      margin-bottom: -30px;
+      padding: 16px 20px 16px 43px;
+      position: relative;
+      z-index: 2;
+
+      &:before {
+        content: $fa-chevron-down;
+        display: block;
+        font-family: $font-family-fontawesome;
+        font-size: 13px;
+        left: 20px;
+        position: absolute;
+        top: 50%;
+        transform: translateY(-50%);
+      }
+
+      &:hover {
+        color: rgba($crm-pure-white-color, .5);
+      }
+
+      + * {
+        box-shadow: $crm-accordion-shadow;
+        display: block;
+        padding-bottom: 20px;
+        padding-top: 20px;
+      }
+    }
+  }
+
+  #Registration .crm-submit-buttons {
+    padding: 8px 20px 20px;
+
+    #_qf_Registration_upload_done-bottom {
+      padding-left: 26px;
+    }
+
+    .fa-check-circle {
+      padding-top: 4px;
+    }
+  }
+
+  .form-layout td.label,
+  .form-layout-compressed td.label,
+  .crm-accordion-body td.label,
+  .crm-form-block td.label {
+    vertical-align: middle;
+
+    label {
+      margin: 0;
+    }
+  }
+
+  .replace-plain {
```

Comment:
This should be created as common style to handle all section as such

example,
```scss
.replace-plain {
  @extend %custom-form-control, %open-sans-font-family;
  height: auto !important;
  line-height: normal !important;
  min-height: 30px;
  width: 300px;
}
```

Reply:
Create this as a separate partial under common directory and include that in common file, so, all such elements are styled automatically

Reply:
I don't know why this is not yet followed.